dataless-classifier/src/main/java/edu/illinois/cs/cogcomp/datalessclassification/classifier/LabelTreeNode.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.datalessclassification.classifier;
+
+import edu.illinois.cs.cogcomp.datalessclassification.hierarchy.TreeNode;
+
+/**
+ * @author shashank
+ */
+
+public class LabelTreeNode extends TreeNode {
+
+    private static final long serialVersionUID = 1L;
+    private String labelName;
+    private String labelDescription;
+
+    public static LabelTreeNode makeBasicNode(String label) {
+        LabelTreeNode node = new LabelTreeNode(label, "", "");
+        return node;
+    }
+
+    /**
+     * Copy Constructor
+     */
+    public LabelTreeNode(LabelTreeNode thatNode) {
+        this(thatNode.getLabel(), thatNode.getLabelName(), thatNode.getLabelDescription());
+    }
+
+    public LabelTreeNode(String label, String labelName, String labelDesc) {
```

Comment:
add javadoc explaining parameters -- having both a 'label' and a 'labelName' parameter is a bit confusing

Reply:
Resolved.